Tinder.API/Program.cs

```diff
@@ -1,10 +1,14 @@
+using Tinder.DAL.DI;
+
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
 // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 
+builder.Services.RegisterDataAccessDependencies(builder.Configuration);

```

Comment:
When you have DI extensions for BLL, move this call to BLL level. API layer should not have reference to DAL